tests/log-uniform-float16.ispc

```diff
@@ -0,0 +1,17 @@
+// rule: skip on arch=x86
+// rule: skip on arch=x86_64
```

Comment:
It's bad idea to have both `x86-64` and `x86_64` here. It needs to be fixed in `run_tests.py`. Please add arch string normalization around line 113 (https://github.com/ispc/ispc/blob/main/run_tests.py#L113), if arch string is `x86_64`, then `x86-64` needs to be used.

Reply:
Changed